backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Services/ConsoleAppHub/ProcessReceivedDataService.cs

```diff
@@ -5,93 +5,109 @@ namespace Squirrel.SqlService.BLL.Services.ConsoleAppHub;
 
 public class ProcessReceivedDataService : IProcessReceivedDataService
 {
-    // TODO: Implement all functions to process data received from ConsoleApp 
+    private readonly ResultObserver _resultObserver;
 
-    /// <summary>
-    /// Just for debugging and demo
-    /// </summary>
-    private Task ShowResult(string clientId, QueryResultTableDTO queryResultTableDTO)
+    public ProcessReceivedDataService(ResultObserver resultObserver)
+    {
+        _resultObserver = resultObserver;
+    }
+    private Task ShowResult(Guid httpId, QueryResultTableDTO queryResultTableDTO)
     {
         Console.WriteLine($"------------------------------------------------------------------");
-        Console.WriteLine($"Successfully recived data from user '{clientId}'");
+        Console.WriteLine($"Successfully recived data from user '{httpId}'");
         Console.WriteLine($"    result:");
         Console.WriteLine(queryResultTableDTO);
         Console.WriteLine($"------------------------------------------------------------------");
         return Task.CompletedTask;
     }
 
-    public async Task AllTablesNamesProcessReceivedDataAsync(string clientId, QueryResultTableDTO queryResultTableDTO)
+    public async Task AllTablesNamesProcessReceivedDataAsync(Guid queryId, QueryResultTableDTO queryResultTableDTO)
```

Comment:
it seems we should decide what the ID is here and in other places (`user` or `client` or `http` or `queryId`)